discovery-provider/default_config.ini

```diff
@@ -27,7 +27,7 @@ session_cookie_secure = false
 ; note that host and port have higher precedence over provider_url
 host = localhost
 port = 8545
-eth_provider_url = http://audius_ganache_cli_eth_contracts:8545
+eth_provider_url = http://localhost:8546
```

Comment:
this doesn't cause issues with docker addressing?

Reply:
might be a docker for mac relic actually, where we needed to address by containe rname instead of just local host

Reply:
this was correct -- but i instead put this in the .env so it's being overridden the same way at least as the ganache_cli for data contracts